Debug/src/.gitignore

```diff
@@ -0,0 +1 @@
+*.o
```

Comment:
We only need one git ignore file at the root directory. I'll bet smart git or eclipse is helping with these. 

You only need one.